packages/block-editor/src/components/rich-text/event-listeners/input-rules.js

```diff
@@ -14,7 +14,7 @@ import {
 	START_OF_SELECTED_AREA,
 } from '../../../utils/selection';
 
-function findSelection( blocks ) {
+export function findSelection( blocks ) {
```

Comment:
Hey @ellatrix 👋 I was wondering if there would be any concerns about exporting this so the native editor can leverage its functionality?